source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
-                  <p>Detailed analysis of the title page and other preliminaries of older printed books and manuscripts is of major importance in descriptive bibliography and the cataloging of printed books. The following elements are suggested as a means of encoding the major features of most title pages for faithful rendition:</p>
+               <div xml:id="manuscriptsAndPrints" type="div3">
+                  <head>Manuscripts and prints</head>
+                  <div xml:id="conditionColophone" type="div4">
+                     <head>Condition and Statement of Production</head>
+                  <p></p>
                   <p>
-                     <specList>
-                        <specDesc key="titlePage"/>
-                        <specDesc key="p"/>
-                        <specDesc key="table"/>
-                        <specDesc key="list"/>
-                        <specDesc key="quote"/>
-                        <specDesc key="lg"/>
+                        <specList>
+                        <specDesc key="condition"/>
+                        <specDesc key="colophone"/>
                      </specList>
-                  </p>
-                  <p>The following example shows the encoding of the title page of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark the line breaks present in the original.</p>
-                  <p>
-                     <figure>
-                        <head/>
-                        <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/text/text-sample371.txt" parse="text"/></egXML>
-                     </figure>
-                  </p>
-                  <p>The physical rendition of title page information is often of considerable importance. One approach to this requirement would be to use the <gi scheme="MEI">rend</gi> element, described in chapter <ptr target="#sharedTextRendition"/> to specify the rendition of each of the components of the title page. Another would be to employ a CSS stylesheet. Finally, a module customized for the description of typographic entities such as pages, lines, rules, etc., bearing special-purpose attributes to describe line-height, leading, degree of kerning, font, etc. could be employed.</p>
+                     </p>
+                  <p>Every physical part of a source is in a certain condition. The element <gi scheme="MEI">condition</gi> may contain plain text but also provides several elements that can be used for structuring the text if necessary (e.g. for linking) in order to describe the state or condition of a source.</p>
```

Comment:
```suggestion
                  <p>The element <gi scheme="MEI">condition</gi> may contain text, or several elements that may be used for structuring the description (e.g. for linking) to accurately describe the physical state or condition of a source.</p>
```

Reply:
Slight re-wording to improve conciseness and clarity. The distinct elements of the first sentence has been worked into the second.